### DISCUSSION TITLE
Codespaces counting towards students time

### DISCUSSION THREAD

**Discussion by Mr-Coxall** [community/community#120764] [GitHub Education]
### Hi everyone,

I have 2 different GitHub organizations that I use for different classes.

On 1 of them (MTHS-ICD2O-1-2023) when students create a Codespace through GitHub Classroom, the time is covered by the organization.
The second (MTHS-ICS3U-2-2023) when student's create a Codespace, it uses up their monthly allotted time, and eventually they are running out.

<img width="1396" alt="codespaces" src="https://github.com/community/community/assets/3744969/d370d6bd-37e2-4790-935a-237956071f5a">

Both have been upgraded to "teams".
Both have "Codespaces in your Classroom" enabled.
I can not think or find anything else different between them.

Is there a setting somewhere that I am missing?
Thanks, Patrick

**Comment by david-wei-01001** [comment]
In your situation with the two GitHub organizations, since both have been upgraded to "teams" and both have "Codespaces in your Classroom" enabled, they should theoretically function in the same way in terms of billing for Codespace usage. The discrepancy where one organization is covering the Codespace time while the other is deducting from the student's monthly allotment is unusual.

Here are some steps you could follow to resolve this issue:

1. **Check Billing Settings**: In the GitHub organization where students' time is being deducted, check the billing settings to ensure that Codespaces are set up to be paid by the organization and not by individual users.

2. **Review Educational Benefits**: Since this is for a classroom setting, make sure that both organizations have the same educational benefits applied. Sometimes, GitHub offers additional benefits for educational institutions, which might need to be manually activated or confirmed.

3. **Inspect Organization Membership Settings**: Ensure that all students are properly set up as members of the organization with the correct privileges. There might be a setting that is applied differently between the two organizations.

4. **GitHub Classroom Settings**: Double-check the GitHub Classroom settings for each organization. There may be an option related to Codespaces that is configured differently.

5. **Compare Codespace Plans**: Look at the specific Codespace plans that are active for each organization. They should be identical, but it's possible there is a difference that's affecting how usage is calculated.

6. **Reach Out to GitHub Support**: Since the issue could be related to billing or specific features of GitHub Classroom, contacting GitHub Support directly may be necessary. They can provide insight into organizational settings and billing details that aren't easily visible or adjustable from the user interface.

7. **Documentation and Change Logs**: Review any available documentation and change logs for recent updates to GitHub Classroom and Codespaces. Sometimes changes to the platform can introduce new settings or affect existing configurations.

8. **Community and Forums**: Check if there are similar cases reported by other users in the GitHub Community Forum or other educational tech forums. Sometimes, other educators may have encountered and solved similar issues.

Given that you've already checked the main features and settings, it may very well be a subtle configuration issue or a difference in how the team plans are set up. If there's a GitHub representative or account manager for educational institutions you can reach out to, they might be able to provide a rapid resolution.

**Comment by Mr-Coxall** [comment]
The solution was in the Org's settings, codespaces, general.
Have to have "Organization Ownership" set.
For some reason my others Orgs were set but this one was not.
![codespaces](https://github.com/community/community/assets/3744969/b44fa41b-ecbe-49b3-9699-77381ed808f6)
